src/eth/monitor.ts

```diff
@@ -109,7 +109,7 @@ async function main() {
       }
     );
   
-    const warningAmount = "1";
+    const warningAmount = config.gasWarningAmount;
```

Comment:
Use config is ok, but need add codes to treat if config do not have gasWarningAmount set.
(like check null and set to default 1, etc)

Also monitor need add an optinal input parameter to manually start monitor with gasWarningAmount, and override config if config has the setting.